fury/actor.py

```diff
@@ -444,6 +444,63 @@ def contour_from_roi(data, affine=None,
     return skin_actor
 
 
+def contour_from_label(data, affine=None,
+                       color=None, opacity=None):
+    """Generate surface actor from a binary labeled Array.
+
+    The color and opacity of individual surfaces can be customized.
+
+    Parameters
+    ----------
+    data : array, shape (X, Y, Z)
+        A labeled array file that will be binarized and displayed.
+    affine : array, shape (4, 4)
+        Grid to space (usually RAS 1mm) transformation matrix. Default is None.
+        If None then the identity matrix is used.
+    color : (N, 3) ndarray
+        RGB values in [0,1]. Default is None.
+        If None then random colors are used.
+    opacity : float or (N,) ndarray
+        Opacity of surface between 0 and 1. Default is None
+        if opacity is numeric, same value is applied to all surfaces.
+        if opacity is None, then 1.0 is used by default for all surfaces.
+
+    Returns
+    -------
+    contour_assembly : vtkAssembly
+        Array surface object displayed in space
+        coordinates as calculated by the affine parameter
+        in the order of their roi ids.
+    """
+
+    unique_roi_id = np.delete(np.unique(data), 0)
+
+    nb_surfaces = len(unique_roi_id)
+
+    unique_roi_surfaces = vtk.vtkAssembly()
+
+    if opacity is None:
+        opacity = np.ones((nb_surfaces, 1)).astype(np.float)
+    elif isinstance(opacity, (float, int)):
+        opacity = np.full((nb_surfaces), opacity)
+    elif opacity.shape != (nb_surfaces,):
+        raise ValueError("Incorrect opacity array shape")
```

Comment:
this block is not needed normally

Reply:
Actually I kept this block in case the user wanted to have the same opacity for all the contour surfaces. So instead of passing an array of opacity values, one can simply pass a number as opacity and it would be converted to an array internally.